src/components/Form/Hooks/useForm.ts

```diff
@@ -53,9 +54,11 @@ export default function useForm<Values = any>(
         scrollToField: (name: OcNamePath, options: ScrollOptions = {}) => {
           const namePath = toArray(name);
           const fieldId = getFieldId(namePath, wrapForm.__INTERNAL__.name);
-          const node: any | null = fieldId
-            ? document.getElementById(fieldId)
-            : null;
+          let node: any | null = null;
+
+          if (canUseDocElement()) {
+            node = fieldId ? document.getElementById(fieldId) : null;
```

Comment:
wouldn't optional chaining document suffice?

Reply:
canUseDocument ensures the window also exists.